feeding/scripts/demo.cpp

```diff
@@ -170,7 +170,8 @@ void demo(
       // ===== IN FRONT OF PERSON =====
       ROS_INFO_STREAM("Move forque in front of person");
 
-      bool tilted = (foodName != "celery");
+      // TODO: Set tilted explcitly for long food items:
+      bool tilted = (foodName == "celery" || foodName == "carrot" || foodName == "bell_pepper" || foodName == "apple");
```

Comment:
Are food properties like "long" stored anywhere?

Reply:
Not right now. Let me add it to `feeding_demo.yaml`!

Reply:
Done! Ready for re-review